tests/node.rs

```diff
@@ -5,9 +5,22 @@
 extern crate wasm_bindgen_test;
 use wasm_bindgen_test::*;
 
+use web_sys::console::error_1;
+use web_sys::console::log_1;
+
 #[wasm_bindgen_test]
 fn simple_dissect_test() {
     let bytestream = "003096e6fc3900309605283888470001ddff45c0002800030000ff06a4e80a0102010a2200012af90017983210058dd58ea55010102099cd00000000";
-    scalpel::dissect_packet(bytestream.to_string());
+    let encap_type = scalpel::ENCAP_TYPE_ETH;
+
+    match scalpel::dissect_packet(encap_type, bytestream.to_string()) {
```

Comment:
We should also add another API something like `dissect_packet_bin` when the data is available as actual binary data. While using this, we realized there's an unnecessary round trip to hex string and back. So see if that can be avoided by defining new API.